src/__tests__/pet.test.js

```diff
@@ -0,0 +1,168 @@
+const { Pet } = require ('../pet.js');
+
+const ERROR_MESSAGE = 'Your pet is no longer alive :(';
+
+let fido;
+beforeEach(() => {
+    fido = new Pet('Fido');
+});
+
+describe ('Pet', () => {
+
+  test('Returns an instance of Pet', () => {
+    expect(fido).toBeInstanceOf(Pet);
+  });
+
+  test('Returns the pets name', () => {
+    expect(fido.name).toBe('Fido');
+  });
+});
+
+describe ('growUp', () => {
+  
+  test ('has an initial age of 0', () => {
+    expect (fido.age).toEqual (0);
```

Comment:
Nice, you could also create a second pet and check the initial age is still 0.

Reply:
You could also move this test into the previous describe block as the initial age is not set by the growUp method.

Reply:
You could also move this test into the previous describe block as the initial age is not set by the growUp method.